R/getKey.R

```diff
@@ -0,0 +1,29 @@
+# getKey.R
```

Comment:
This is not a "script" but a function. Scripts should go into `./inst/scripts`.

Reply:
I removed the function. Thanks